practice/pig-latin.py

```diff
@@ -1,58 +1,122 @@
 import string
 
-"""
-Translates english into pig-latin
+""" Translates english into pig-latin.
 Note: I still have not figured out to solve for punctuation that begins a word as of yet.
-Future Goal #1: Solve for Capital and Punctuation on one word [e.g. No.]
-Future Goal #2: Solve for apostraphes
-Future Goal #3: Solve for punctuation that begins a word, [e.g. (Sam) "Bob" or 'Tiffany' 'Tis]
-Future Goal #4: Solve for capitals in the middle of a word, [e.g. McDonalds]
-Future Goal #5: Solve for multiple punctuation [e.g. "Is that butter?" or (Don't turn off the lights.)]
+Future Goal #1: Solve for punctuation that begins a word, [e.g. (Sam) "Bob" or 'Tiffany' 'Tis]
+Future Goal #2: Solve for multiple punctuation [e.g. "Is that butter?" or (Don't turn off the lights.)]
 Note: With these goals, I'd like to avoid ruining punctuation around short words such as I and My.
+EDIT 12-July-2016: Altering code to be a series of functions, solved for some apostrophes and capitals.
+EDIT 12-July-2016: Added list of words in English that start with Y-vowel for higher accuracy.
 """
 
-# 1. Setup
-FIRST_VOWEL = 'a e i o u'.split()
-LATER_VOWEL = 'a e i o u y'.split()
+
+# 1. Setup Define/Functions
+
+
+FIRST_VOWEL = 'a e i o u A E I O U'.split()
```

Comment:
We'll go over this more later, but I suggest having variable names parallel the structure within. So for variables that contain lists, have the variable name be plural as a hint that there are multiple items inside.